code/stable/hghc/SRS/HTML/HGHC_SRS.html

```diff
@@ -153,6 +157,17 @@ <h2>Table of Symbols</h2>
         </div>
       </div>
     </div>
+    <div id="Sec:Intro">
+      <div class="section">
+        <h1>Introduction</h1>
+        <p class="paragraph">
+          Heat transfer through the cladding of a nuclear fuel element influences performance and safety. Engineers therefore rely on dependable calculations of the heat transfer coefficients used for simulating the temperature. This document describes the requirements of a program called HGHC (<span title="HGHC">HGHC</span>).
+        </p>
+        <p class="paragraph">
+          The following section provides an overview of the Software Requirements Specification (<span title="software requirements specification">SRS</span>) for HGHC. This section explains the purpose of this document, the scope of the requirements, the characteristics of the intended reader, and the organization of the document.
```

Comment:
This roadmap is not accurate.  None of the things listed are actually covered in the next section.